packages/documentation/src/stories/foundation/layout/Columns/columns.stories.ts

```diff
@@ -0,0 +1,252 @@
+      <div class="col-1 ${args.offsetItem}">.col-1 .${args.offsetItem}</div>
+    </div>
+  `,
+};
+
+export const ColumnBreakExample: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col-3">.col-3</div>
+      <div class="col-3">.col-3</div>
+
+      <!-- Force next columns to break to new line -->
+      <div class="w-100"></div>
+
+      <div class="col-3">.col-3</div>
+      <div class="col-3">.col-3</div>
+    </div>
+  `,
+};
+
+export const ResetOffsetExample: Story = {
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      ${story(args, context)}
+      <p class="mt-regular"><small>Resize the browser window to see changes.</small></p>
+    `,
+  ],
+  render: () => html`
+    <div class="row">
+      <div class="col-sm-5 col-md-6">.col-sm-5 .col-md-6</div>
+      <div class="col-sm-5 offset-sm-2 col-md-6 offset-md-0">
+        .col-sm-5 .offset-sm-2 .col-md-6 .offset-md-0
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-6 col-md-5 col-lg-6">.col-sm-6 .col-md-5 .col-lg-6</div>
+      <div class="col-sm-6 col-md-5 offset-md-2 col-lg-6 offset-lg-0">
+        .col-sm-6 .col-md-5 .offset-md-2 .col-lg-6 .offset-lg-0
+      </div>
+    </div>
+  `,
+};
+
+export const MarginUtilitiesExample: Story = {
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      ${story(args, context)}
+      <p class="mt-regular"><small>Resize the browser window to see changes.</small></p>
+    `,
+  ],
+  render: () => html`
+    <div class="row">
+      <div class="col-md-4">.col-md-4</div>
+      <div class="col-md-4 ms-auto">.col-md-4 .ms-auto</div>
+    </div>
+    <div class="row">
+      <div class="col-md-3 ms-md-auto">.col-md-3 .ms-md-auto</div>
+      <div class="col-md-3 ms-md-auto">.col-md-3 .ms-md-auto</div>
+    </div>
+    <div class="row">
+      <div class="col-auto me-auto">.col-auto .me-auto</div>
+      <div class="col-auto">.col-auto</div>
+    </div>
+  `,
+};
+
+export const StandaloneColumnExample: Story = {
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      <div class="standalone-columns">
+        ${story(args, context)}
+        <p class="mt-regular"><small>Resize the browser window to see changes.</small></p>
+      </div>
+    `,
+  ],
+  render: () => html`
+    <div class="col-3 p-3 mb-2">.col-3: width of 25%</div>
+
+    <div class="col-md-9 p-3">.col-md-9: width of 75% above md breakpoint</div>
+  `,
+};
+
+export const ColumnWrapping: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col-9">.col-9</div>
+      <div class="col-4">
+        .col-4
+        <br />
+        Since 9 + 4 = 13 &gt; 12, this 4-column-wide div gets wrapped onto a new line as one
+        contiguous unit.
+      </div>
+      <div class="col-6">
+        .col-6
+        <br />
+        Subsequent columns continue along the new line.
+      </div>
+    </div>
+  `,
+};
```

Comment:
Would be nice to have a interactive example here as well.
We could, for example make the class of the first column changeable by a control, so users can change it's width and see how this changes the breaking behavior.